tests/functional/data/configured-endpoint-urls/test-cases/profile-tests.json

```diff
@@ -0,0 +1,426 @@
+{
```

Comment:
Let's also make sure there is a `feature` changelog as part of this PR so the CLI will have a changelog for configurable endpoints when it is released